andela/mrm_api
# #157425740  Set up test coverage

#### What does this PR do?
- This PR sets the configurations for test coverage for `mrm_api`.

#### How should this be manually tested?
 - To run tests and observe test coverage for various versions of python . Run the command below.
 ```
 tox
 ```
 - To run  and check for test coverage. Run the command below:
 ```
 coverage run -m pytest
 ```
 - To obtain coverage report. Run the command below:

 ```
 coverage report
 ``` 
 - To obtain html browser report. Run command below:
 ```
 coverage html
 ```

 `After this,a folder titled html_coverage_report will be generated. Open it and copy the path  of index.html and paste it in your browser.`
 
#### Any background context you want to provide?
- Test coverage is an important aspect to check how well your code has been tested. When tests are written it is imperative that `test coverage` is checked.
- Something key to note, files which have no tests written for them are set to default to `100%` , do ensure test s are written for all modules in order to generate the right test coverage.
#### What are the relevant pivotal tracker stories?
- #157425740
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/30471763/40054513-c0ad97d8-584d-11e8-87f0-c97dbf12763a.png)
![image](https://user-images.githubusercontent.com/30471763/40054545-e1648482-584d-11e8-95c8-81bf4b516f60.png)
